video/out/gpu/video_shaders.c

```diff
@@ -679,9 +682,18 @@ static void pass_tone_map(struct gl_shader_cache *sc,
     if (opts->compute_peak >= 0)
         hdr_update_peak(sc, opts);
 
-    // Always hard-clip the upper bound of the signal range to avoid functions
-    // exploding on inputs greater than 1.0
-    GLSLF("vec3 sig = min(color.rgb, sig_peak);\n");
+    // Experimental #HDR on #macOS        
+    if (opts->curve == TONE_MAPPING_HDR_PASSTHROUGH && opts->curve == TONE_MAPPING_HDR_SCALE)
+    {
+        // Disable any clipping on HDR content
+        GLSLF("vec3 sig = color.rgb;\n");
```

Comment:
Shouldn't you still clip to the new `sig_peak` of 3.0?

Reply:
Yeah, I do that in `pass_tone_map`.